modules/math/src/main/java/com/opengamma/strata/math/impl/statistics/descriptive/ExponentiallyWeightedInterpolationQuantileMethod.java

```diff
@@ -160,30 +157,30 @@ private QuantileResult quantileFromIndexRunningWeight(
     }
     impacts[0] = 1 - alpha;
     impacts[1] = alpha;
-    return QuantileResult.of((1 - alpha) * s[index] + alpha * s[index + 1], indices, DoubleArray.ofUnsafe(impacts));    
+    return QuantileResult.of((1 - alpha) * s[index] + alpha * s[index + 1], indices, DoubleArray.ofUnsafe(impacts));
   }
-  
+
   /**
    * Computes expected shortfall.
-   * @param index  the index from which the ES should be computed
-   * @param runningWeight  the running weight up to index
-   * @param isExtrapolated  flag indicating if value should be extrapolated (flat) beyond the last value
-   * @param s  the sorted sample
-   * @param w  the sorted weights
-   * @param order  the order of the sorted sample in the unsorted sample
-   * @param level  the level at which the ES should be computed
+   *
+   * @param index the index from which the ES should be computed
```

Comment:
Two spaces between parameter name and comment.
The same comments for the other parameters.